apps/Launcher/syncwidget.cpp

```diff
@@ -225,8 +225,11 @@ void SyncWidget::setSceneFiles(QMap<QString, QString> sceneFiles) {
         const QString& sceneName = keys[i];
 
         QCheckBox* checkbox = new QCheckBox(sceneName);
-        checkbox->setChecked(true);
-
+        checkbox->setChecked(false);
```

Comment:
Okay.  Do we want to keep it to be explicit, or rely on anyone reading it knowing that?